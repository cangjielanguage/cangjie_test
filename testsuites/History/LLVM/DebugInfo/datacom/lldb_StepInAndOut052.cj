// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_StepInAndOut052_expected.txt ../basecase/037_debuginfun_lambda.cj
// EXEC: cjc -g 037_debuginfun_lambda.cj
// EXEC: python3 DatacomTest.py lldb_StepInAndOut052_expected.txt



