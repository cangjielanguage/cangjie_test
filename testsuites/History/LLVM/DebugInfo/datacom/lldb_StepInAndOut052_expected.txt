lldb,lldb 037_debuginfun_lambda,037_debuginfun_lambda
b,b 8,Breakpoint
b,b 9,037_debuginfun_lambda.cj:10
b,b 10,037_debuginfun_lambda.cj:10
b,b 11,037_debuginfun_lambda.cj:12
b,b 12,037_debuginfun_lambda.cj:12
r,r,-> 8
s,s,-> 12
s,s,-> 10
finish,finish,-> 12
n,n,-> 13
cj_plocals,cj_plocals,lamdba001 = 20